terraform/environments/mlra/application_variables.json

```diff
@@ -1,7 +1,8 @@
 {
   "accounts": {
     "development": {
-      "example_var": "dev-data"
```

Comment:
Other good things to add in the variables are things like AMI/snapshot IDs or instance sizes, things that will likely change between environments.

You can then reference this in the terraform code for example - 

```
local.application_data.accounts[local.environment].example_var
```
And it will pick up the relevant value for the environment